scripts/qa_crowbarsetup.sh

```diff
@@ -3647,21 +3645,17 @@ function oncontroller_magnum_service_setup
             --property os_distro=fedora-atomic $service_image_name
     fi
 
+    local service_sles_image_name=openstack-magnum-k8s-image
     if ! openstack image list --f value -c Name | grep -q "^${service_sles_image_name}$"; then
-        local ret=$(wget -N --progress=dot:mega "$service_sles_image_url" 2>&1 >/dev/null)
-        if [[ $ret =~ "200 OK" ]]; then
-            echo $ret
-        elif [[ $ret =~ "Not Found" ]]; then
-            complain 73 "SLES magnum image not found: $ret"
-        else
-            complain 74 "failed to retrieve SLES magnum image: $ret"
-        fi
+        safely zypper -n install ${service_sles_image_name}-${arch}
```

Comment:
Hmmm this will be run on Controller node, but below you are downloading this image from admin node?